src/main/java/com/iexec/worker/executor/TaskExecutorService.java

```diff
@@ -104,9 +111,30 @@ public boolean canAcceptMoreReplicates() {
                 });
     }
 
+
+    public void tryToContribute(ContributionAuthorization contributionAuth) {
```

Comment:
I think it would be better to replace this method by an "isReadyToContribute()" method and use it in the subscriptionService so the latter stays the entrypoint that explicitly creates replicates (even when recovering)

Reply:
Yes, we are going to tune that in a second time